addon/mixins/transition-mixin.js

```diff
@@ -1,7 +1,14 @@
+import { alias } from '@ember/object/computed';
+import Mixin from '@ember/object/mixin';
+import RSVP from 'rsvp';
+import { computed } from '@ember/object';
+import { run } from '@ember/runloop';
+import $ from 'jquery';
+import { A } from '@ember/array';
+import { dasherize } from '@ember/string';
 import Ember from 'ember';
 const {
-  Mixin, RSVP, computed, run, testing, $, A,
-  String: { dasherize }
+  testing
 } = Ember;
```

Comment:
We can remove Ember.testing as well. Can you do that, please?
You can check paper-dialog source.

Reply:
Hmm, it's probably not going to be very easy because we use ember testing in the module scope. Where should we call getOwner on?

Reply:
Don't know - need `this` to be created by the container. I see `rAF` & `cAF` both being exported from addon/mixin but not from the `app` dir  - I think it means no one can consume those outside the `ember-css-transitions`. We could hang those as private properties off of the mixin, therefore allow use to get configuration from container through Ember.applicationInstance.

Reply:
There is also an `https://github.com/html-next/ember-raf-scheduler` that integrates with the Ember runloop. https://github.com/html-next/vertical-collection uses it.

Reply:
Do we leave it as is and wait until they deprecate this or ?